core/lib/spree/core/promotion_configuration.rb

```diff
@@ -57,6 +57,14 @@ class PromotionConfiguration < Spree::Preferences::Configuration
       #   Spree::PromotionHandler::Coupon.
       class_name_attribute :coupon_code_handler_class, default: 'Spree::PromotionHandler::Coupon'
 
+      # Allows providing a different coupon code handler.
+      # @!attribute [rw] coupon_code_handler_class
+      # @see Spree::PromotionHandler::Coupon
+      # @return [Class] an object that conforms to the API of
+      #   the standard coupon code handler class
+      #   Spree::PromotionHandler::Coupon.
```

Comment:
Copy paste error in the rdoc ;)

Reply:
Thank you, this was true for both promotion configuration classes. Fixed!